.env.example

```diff
@@ -1,17 +1,20 @@
 # rename this file to .env and supply the values listed below
 # also make sure they are available to the build tool (e.g. Netlify)
-# warning: variables prefixed with GATSBY_ will be made available to client-side code
+# warning: variables prefixed with NEXT_PUBLIC_ will be made available to client-side code
 # be careful not to expose sensitive data (e.g. your Algolia admin key)
-# ALGOLIA_ADMIN_KEY=insertValue
+
+# Algolia environment (app ID, search key and base search index name required for search)
+# NEXT_PUBLIC_ALGOLIA_APP_ID=insertValue
```

Comment:
do we need the `ALGOLIA` vars to be prefixed by `NEXT_PUBLIC`?

Reply:
We do, yes... these are consumed by the component when accessing the index. None of these variables are sensitive (just read-access to the index), and is parity to the canonical site where we use the `GATSBY_` prefix.